### DISCUSSION TITLE
*:logstash-* does not contain any of the following.....

### DISCUSSION THREAD

**Discussion by DebianGuru** [Security-Onion-Solutions/securityonion#7276] [Unsupported Versions] [answered]
I recently re-installed SO 2.3.100.  I have a standalone setup.  I had some custom dashboards on my previous SO install that I wanted to recreate.  My previous SO install had been up for some time and has gone through several upgrades, so I'm not sure what version it started at.

Now when I try to recreated my old dashboards by adding new visualizations from library I often get the following error:
`The index pattern *:logstash-* does not contain any of the following compatible field types: number, boolean, date, ip, or string`
I also try to create new visualizations and often see 

```
Available fields
0
There are no available fields that contain data.
Try:
Extending the time range
```
when I select the index pattern for *:logstash-*.  It seems to me (with my limited understanding) that logstash data is not being stored with that pattern.  I'm trying to remember, but I think the old version used something like *:so-logstash-*, but I don't remember. 

Any idea what's going on?  Can I change this so all the visualizations from the library will work again?  Thanks.

**Comment by dougburks** [comment, marked as answer]
It sounds like your custom dashboards were from an older Security Onion 16.04 installation. Security Onion 2 uses `*:so-*` as its default index pattern:
![image](https://user-images.githubusercontent.com/1659467/155157021-fe3c33e2-8b49-4f2b-8083-cc0a412b4b47.png)

**Reply by DebianGuru** [reply]
Doug,
Thanks for the reply, but my previous SO was indeed 2.3.?? (then updated through 2.3.100).  I was not trying to restore some kind of saved dashboard, rather than just duplicating it's function based upon the tools from 2.3.100.  
I still show a lot of things in this fresh install that refer to the logstash when I try to add any visualizations from the library (these are canned items that came with a fresh SO 2.3.100 install).  From what you're saying, a fresh 2.3.100 install (as I just did) shouldn't still be pointing to  :logstash-

**Reply by dougburks** [reply]
Security Onion 2.3.100 includes the legacy dashboards from Security Onion 16.04 just in case folks are doing an in-place upgrade and need to see their old data. These dashboards are named with the `z16.04` prefix and their searches use the `*:logstash-*` indices since those are the indices we used in 16.04. Going forward, most of our new data should be using the `*:so-*` indices and our new dashboards named with the `Security Onion` prefix should be using those indices:
https://docs.securityonion.net/en/2.3/kibana.html#dashboards

**Reply by DebianGuru** [reply]
Doug, 
Thanks so much for the info.  That makes sense.  I wondered how those dashboards magically appeared.  I would never have thought to look for documentation for 2.3.100 (current version) in the 16.04 documentation.  Perhaps (if it isn't already) you may want to add some reference to that in the 2.3.100 documentation also for people in my boat.

**Reply by dougburks** [reply]
I've updated the current documenation at https://docs.securityonion.net/en/2.3/kibana.html#dashboards from:

> We’ve included the old 16.04 dashboards in case you performed an in-place upgrade and have any old 16.04 data. These dashboards are named with the z16.04 prefix and will only show old 16.04 data. The new Security Onion 2 dashboards are all named with the Security Onion prefix and they should be used for any new data going forward. 

to:

> We've included the old 16.04 dashboards in case you performed an in-place upgrade and have any old 16.04 data. These dashboards are named with the z16.04 prefix and will only show old 16.04 data from the old *:logstash-* indices. The new Security Onion 2 dashboards are all named with the Security Onion prefix and they should be used for any new data stored in the new *:so-* indices.

**Reply by DebianGuru** [reply]
Doug, thanks for the updated documentation.  I hope it helps others.  Security Onion Solutions, LLC is a first rate company with the best support (even the community support).  I have no reservations at recommending your solution to anyone.  We do business with all the big companies, and you guys outshine them by a mile.  Kudos to you and your staff.

**Reply by dougburks** [reply]
Thanks for your kind words!
